PhysicsTools/PatUtils/interface/SmearedJetProducerT.h

```diff
@@ -234,7 +243,7 @@ class SmearedJetProducerT : public edm::stream::EDProducer<> {
                     }
 
                     double dPt = jet.pt() - genJet->pt();
-                    smearFactor = 1 + (jer_sf - 1.) * dPt / jet.pt();
+                    smearFactor = 1 + _nomVar*(jer_sf - 1.) * dPt / jet.pt();
```

Comment:
I'm probably missing something.
What is the purpose of this "101" shift?
For +/-101 we get the same NOMINAL jet_sf value and you will smear by the full ```+/- |jet_sf -1|```
while the systematics on the JER can be smaller or larger

Looking at 
https://twiki.cern.ch/twiki/bin/view/CMS/JetResolution#JER_Scaling_factors_and_Uncertai
for eta 0 to 0.5 JER SF is 1.122 +-0.026 or "jer_sf - 1 = 0.122 +-0.026".

So, smearing by +-0.122 in this case is not meaningful, but maybe there is some other use case outside the nominal definition of JER.
Please clarify.